packages/shorebird_cli/lib/src/commands/build_command.dart

```diff
@@ -43,6 +50,13 @@ class BuildCommand extends ShorebirdCommand
       return ExitCode.software.code;
     }
 
+    final flutterValidationIssues = await _flutterValidator.validate();
```

Comment:
We could probably also create a `ValidateFlutterVersionMixin` on `ShorebirdCommand` to simplify this a bit further so then each command could use the mixin like:

```dart
class BuildCommand extends ShorebirdCommand with ValidateFlutterVersionMixin {
  Future<int> run() async {
    ...
    warnOnFlutterVersionMismatch();
  }
}
```

Reply:
Great idea, done